app/controllers/BaseController.java

```diff
@@ -126,6 +126,16 @@ protected UserSession getAuthenticatedResearchOrAdminSession(Study study) {
         }
         throw new UnauthorizedException();
     }
+    
+    protected UserSession getAuthenticatedResearchSession(Study study) {
```

Comment:
Naming: Should it be ResearchSession or ResearcherSession? Also note that above, there are two identical methods named ResearcherOrAdminSession and ResearchOrAdminSession...